CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.0.1
+
+**Release date:** 2023-07-11
+
+This release includes flux2 [v2.0.1](https://github.com/fluxcd/flux2/releases/tag/v2.0.1).
+
+Improvements:
+- Update dependencies
+  [#519](https://github.com/fluxcd/terraform-provider-flux/pull/519)
+- Bump fluxcd/flux2 from 0.41.0 to 2.0.0
```

Comment:
We decided to not include GH actions bumps in changelog as these could confuse users and have no relevance to the release.